zerver/tests/test_bots.py

```diff
@@ -414,6 +416,39 @@ def test_bot_add_subscription(self) -> None:
         # Test runner automatically redirects all sent email to a dummy 'outbox'.
         self.assert_length(mail.outbox, 0)
 
+    def test_bot_add_user_group(self) -> None:
+        hamlet = self.example_user("hamlet")
+        self.login("hamlet")
+        params = {
+            "name": "support",
+            "members": orjson.dumps([hamlet.id]).decode(),
+            "description": "Support team",
+        }
+
+        self.client_post("/json/user_groups/create", info=params)
```

Comment:
We can use existing user group of hamletcharacters here, there is no need to create a new group.